ReleaseProcess.md

```diff
@@ -3,20 +3,23 @@
 
 1. Update the versions of the subprojects you are using to the latest versions
    of master using `update_versions_to_latest_master.py` (or manually update
-   versions.py to specific versions) and commit your change.
-2. Tag the release and build the release log by running 
+   versions.py to specific versions).
+2. Stage changes with `git add .` and commit and push these changes. At this point, `git status` may show some submodules as 
+   having modified content, not staged for commit. (This will happen if submodules, e.g. models, have changed.) 
```

Comment:
Maybe "This will happen if submodules of submodules, e.g. models. have changed."